src/services/keycloak/setup.ts

```diff
@@ -142,10 +132,6 @@ export class KeycloakSetupService {
       if (e.response.status === 404) {
         return;
       }
-      if (this.isDevelopment) {
```

Comment:
It won't be caught with ErrorBoundary.  I suggest leaving as is.
These changes do not apply to the target commit.

Reply:
The exception gets handled [here](https://github.com/eclipse/che-dashboard/blob/cc2e9fbd672993d9ef853cc2114b549aab4accc9/src/index.tsx#L35) and then the dashboard continues to initialize. I tested it running dashboard in dev mode either locally and in Che. I may miss something though, if you think so please elaborate on this.